dashboard/app/models/user.rb

```diff
@@ -1329,6 +1329,26 @@ def to_csv
     User.csv_attributes.map {|attr| send(attr)}
   end
 
+  # Format user information for the JSON API
+  def summarize
```

Comment:
Compare to output at https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/helpers/section_api_helpers.rb#L494-L533